v2/src/css/custom.css

```diff
@@ -519,3 +519,45 @@ html[data-theme="dark"] .markdown blockquote > *:last-child {
     bottom: 144px !important;
   }
 }
+
+/* Styles for collapsible elements */
+details {
+  position: relative;
+}
+
+details > summary {
+  list-style-type: none;
+  position: relative;  
+  display: flex;
+  align-items: center;
+  background-color: var(--ifm-pre-background);  
+  padding: 14px 20px;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+details > summary::before {
+  content: "";
+  position: relative;
+  box-sizing: border-box;
+  display: inline-block;
+  background: url(/img/icon-plus.png) center;
+  background-size: contain;
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  margin-right: 12px;
+}
+
+details[open] {
+  padding-bottom: 20px;
+  border-bottom: 1px solid var(--ifm-toc-border-color);;
```

Comment:
The detail component in the design has a border top and bottom in both open and closed states (Its very light so it is easy to miss in the design)